compiler/z/env/OMRCPU.cpp

```diff
@@ -266,10 +266,10 @@ OMR::Z::CPU::getSupportsArch(Architecture arch)
    return _supportedArch >= arch;
    }
 
-bool
+void
 OMR::Z::CPU::setSupportsArch(Architecture arch)
    {
-   return _supportedArch = _supportedArch >= arch ? _supportedArch : arch;
+   _supportedArch = _supportedArch >= arch ? _supportedArch : arch;
    }
```

Comment:
This `setSupportsArch` method seems a bit odd to me.  This is one where returning a boolean value to indicate whether the method had an effect might make sense, although it's currently returning something that will always be implicitly cast to true, I think.  I see it's never actually used, so perhaps it should be removed?

Reply:
Yes, looks like I just forgot to remove the `return`

Reply:
No, you had removed the `return`.  What I find odd is that the method might not do what is requested.  For instance, if I try to call `setSupportsArch(z9)` when the default `z10` is in effect, the method won't do anything.  Having it return a boolean that indicates whether the method had an effect might be useful.

But as I said, the method is never used, so maybe it should be removed.